examples/reference-entity-model/steps/merging/merge-customers.step.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name" : "merge-customers",
+  "stepDefinitionName" : "default-merging",
+  "stepDefinitionType" : "MERGING",
+  "sourceQuery" : "cts.collectionQuery('matched-customers')",
+  "targetEntity" : "Customer",
+  "sourceDatabase" : "data-hub-FINAL",
+  "collections" : [ "merged-customer", "merge-customers" ],
+  "mergeOptions" : {
+    "propertyDefs" : {
+      "properties" : [ {
+        "namespace" : "",
+        "localname" : "customerId",
+        "name" : "customerId"
+      }, {
+        "namespace" : "",
+        "localname" : "name",
+        "name" : "name"
+      } ]
+    },
+    "algorithms" : {
+      "stdAlgorithm" : {
+        "timestamp" : {
+          "path" : ""
```

Comment:
Is this left blank on purpose? The createdOn timestamp in Explorer for a merged document might show up "Invalid"

Reply:
It's how it was copied over - the conversion process isn't looking to change any matching/merging options, it just copies them over as-is. 

Note though that "createdOn" is driven by a document metadata key. This timestamp is used for a different purpose.